maps/_DeepForest/map/_Deep_Forest.dmm

```diff
@@ -12542,6 +12542,9 @@
 /obj/structure/closet/crate,
 /turf/simulated/floor/tiled/derelict/white_small_edges,
 /area/nadezhda/outside/one_star)
+"Vx" = (
+/turf/simulated/wall/untinted/onestar_reinforced,
+/area/asteroid/rogue)
```

Comment:
why is the area this?

Reply:
Greyson
![image](https://user-images.githubusercontent.com/10091683/187025842-34eab71e-5558-4143-a2fe-63175429f20c.png)

Reply:
Ah, fair enuff